setup.py

```diff
@@ -92,30 +79,24 @@ def run_tests(self):
     version=version,
     description=__doc__,
     long_description=readme + '\n\n' + history,
-    keywords='invenio TODO',
+    keywords='invenio minter pidstore autoincrement',
     license='GPLv2',
     author='CERN',
     author_email='info@inveniosoftware.org',
     url='https://github.com/inveniosoftware/invenio-sequencegenerator',
-    packages=[
-        'invenio_sequencegenerator',
-    ],
+    packages=packages,
     zip_safe=False,
     include_package_data=True,
     platforms='any',
     entry_points={
-        'console_scripts': [
-            'bibtex = invenio_sequencegenerator.scripts.bibtex:main',
-        ],
-    },
-    install_requires=requirements,
-    extras_require={
-        'docs': [
-            'Sphinx>=1.3',
-            'sphinx_rtd_theme>=0.1.7'
+        'invenio_i18n.translations': [
+            'messages = invenio_sequencegenerator',
```

Comment:
Just make sure that you include catalog in `invenio_sequencegenerator/translations`.

```
mkdir invenio_sequencegenerator/translations
python setup.py extract_messages
python setup.py init_catalog -l fr
```